config/environments/production.rb

```diff
@@ -95,8 +95,8 @@
 
   config.action_mailer.default_url_options = { host: 'https://managing-registers.cloudapps.digital' }
 
-  config.register_phase = :beta
-  config.register_url = 'beta.openregister.org/load-rsf'
+  config.register_phase = 'beta'
+  config.register_url = 'beta.register.gov.uk/load-rsf'
```

Comment:
Should this be 'register.gov.uk/load-rsf'? I also think we should keep it as `beta.openregister.org/load-rsf` because we currently don't allow POST methods via the CDN.